app/editor/src/components/upload/Upload.tsx

```diff
@@ -37,88 +39,93 @@ export const Upload: React.FC<IUploadProps> = ({
   onSelect,
   onDownload,
   onDelete,
+  contentType,
+  stream,
   ...rest
 }) => {
   const { isShowing, toggle } = useModal();
-  const fileRef = React.useRef<HTMLInputElement>(null);
-
+  const { values, setFieldValue } = useFormikContext<IContentForm>();
   const [file, setFile] = React.useState<IFile>();
-
-  /** Duration / metadata WIP */
-  // const [duration, setDuration] = useState(0);
-  const reader = new FileReader();
   const fileName = generateName(file) ?? generateName(initFile);
-
+  console.error(contentType === ContentTypeName.Snippet, 'contentType');
```

Comment:
Remove

Reply:
Ugh, I did a ctrl + f for console.log lol